python/pyiceberg/catalog/rest.py

```diff
@@ -225,15 +228,18 @@ def url(self, endpoint: str, prefixed: bool = True, **kwargs) -> str:
         return url + endpoint.format(**kwargs)
 
     def _fetch_access_token(self, credential: str) -> str:
-        client_id, client_secret = credential.split(":")
+        if SEMICOLON not in credential:
+            raise ValueError("Expected semicolon in the credential to separate the client id and secret.")
```

Comment:
In Java, we just use the whole string as the client secret and set the client ID to empty string.

Reply:
I like that, updated! 👍🏻